enterprise/ha/src/main/java/org/neo4j/kernel/ha/cluster/modeswitch/HighAvailabilityModeSwitcher.java

```diff
@@ -369,7 +369,7 @@ public void run()
                 }
                 catch ( HighAvailabilityStoreFailureException e )
                 {
-                    userLog.error( "UNABLE TO START UP AS SLAVE: %s", e.getMessage() );
+                    userLog.error( "UNABLE HEADER_TO START UP AS SLAVE: %s", e.getMessage() );
```

Comment:
nope

Reply:
Did not anticipate shouting, reverting...